segment/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "segment",
```

Comment:
Can we give this a less generic name? I suppose in the future we'll want to add more Segment integrations. My suggestion would be `segment-event-notification` if you don't have a concrete use case

Reply:
Updated to `segment-event-notification`.